tests/interpreters/test_arith_interpreter.py

```diff
@@ -238,6 +239,85 @@ def test_cmpi(
     assert ret[0] == fn(lhs_value, rhs_value)
 
 
+def o(x: float, y: float) -> bool:
+    return not isnan(x) and not isnan(y)
+
+
+def u(x: float, y: float) -> bool:
+    return isnan(x) or isnan(y)
```

Comment:
let's inline these below? might as well just have one value for the nan comparison, no need for helper functions IMO